vllm/worker/model_runner.py

```diff
@@ -928,10 +928,10 @@ def profile_run(self) -> None:
         torch.cuda.synchronize()
         return
 
-    def remove_all_loras(self) -> bool:
+    def remove_all_loras(self):
```

Comment:
verified with callers?

Reply:
yeah return value is not used by anything